Foundation/URLSession/NativeProtocol.swift

```diff
@@ -0,0 +1,754 @@
+// Foundation/NSURLSession/_NativeProtocol.swift - NSURLSession & libcurl
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+// -----------------------------------------------------------------------------
+///
+/// This file has the common implementation of Native protocols like HTTP,FTP,Data 
+/// These are libcurl helpers for the URLSession API code.
+/// - SeeAlso: https://curl.haxx.se/libcurl/c/
+/// - SeeAlso: NSURLSession.swift
+///
+// -----------------------------------------------------------------------------
+
+import CoreFoundation
+import Dispatch
+
+internal let enableLibcurlDebugOutput: Bool = {
+    return  (ProcessInfo.processInfo.environment["URLSessionDebugLibcurl"] != nil)
```

Comment:
The parenthesis around these values appear to have no value. Can you remove them, and get rid of the double-space after the `return` keyword?